data/src/main/java/com/antosik/benchproject/data/popular/movies/repo/PopularMoviesDataRepository.kt

```diff
@@ -13,7 +13,7 @@ internal class PopularMoviesDataRepository(
     override suspend fun getPopularMoviesList() = runCatching {
         val remoteMovies = apiService.getPopularMovies()
         dao.insertMovies(remoteMovies.toEntity())
-    }.run {
+    }.let {
```

Comment:
`let` is exposing the result through `it`, you should probably use it.

Reply:
Ok, but `let` return `Result<Unit>` and I have no idea what I am supposed to do with it except this:
![image](https://user-images.githubusercontent.com/57491794/225307543-3d3525d7-14e0-46b7-94b3-e9d67d2ad8c4.png)
and this did not help me to return the data. Because when we talked about this we focused on statements that are always invoked after inserting data from API into the database

Reply:
To be honest I don't understand 😄 let's talk about it tommorow

Reply:
I agree 😄

Reply:
I changed it